### PR TITLE
Update deploy docs about kinds of deploys, schedules, who does them

### PR DESCRIPTION
**Pull request by zachmargolis**
Some updates based on our conversation at eng huddle yesterday

### CODE REVIEW

File: _articles/appdev-deploy.md [hunk 1]
```diff
@@ -3,8 +3,46 @@ title: "Deploying new IDP and PKI code"
...
+| ---- | ---- | ---- | --- |
+| **Full Deploy** |  The normal deploy, releases all changes on the `main`  branch to production. | Every 2 weeks | [AppDev Deploy Manager rotation][deployer-rotation] |
+| **Patch Deploy** | A deploy that cherry-picks particular changes to be deployed | For urgent bug fixes | [AppDev Deploy Manager rotation][deployer-rotation], or engineer handling the urgent issue |
+| **Off-Cycle/Mid-Cycle Deploy** | Releases all changes on the `main` branch, sometime during the middle of a sprint | As needed, or if there are too many changes needed to cleanly cherry-pick as a patch | The engineer that needs the changes deployed |
+
+[deployer-rotation]: https://login-gov.app.opsgenie.com/settings/schedule/detail/7f1b7c07-e6de-4990-8b59-01cc4c681542
+
+### Communications
+
+Err on the side of overcommunication about deploys: the steps are the are
```
Comment (solipet) [inline _articles/appdev-deploy.md:40]: `are the are`?
```suggestion
Err on the side of overcommunication about deploys: the steps as they are
```

File: _articles/appdev-deploy.md [hunk 2]
```diff
@@ -3,8 +3,46 @@ title: "Deploying new IDP and PKI code"
...
+happening. Especially overcommunicate about off-cyclel/mid-cycle deploys, as
```
Comment (solipet) [inline _articles/appdev-deploy.md:41]:
```suggestion
happening. Especially overcommunicate about off-cycle/mid-cycle deploys, as
```

File: _articles/appdev-deploy.md (hunk 1 above)
Comment (aduth) [inline _articles/appdev-deploy.md:40]: >[...] the steps are the are [...]

Not clear to me what this is trying to say.

File: _articles/appdev-deploy.md (hunk 2 above)
Comment (aduth) [inline _articles/appdev-deploy.md:41]:
```suggestion
happening. Especially overcommunicate about off-cycle/mid-cycle deploys, as
```

File: _articles/appdev-deploy.md [hunk 3]
```diff
@@ -3,8 +3,46 @@ title: "Deploying new IDP and PKI code"
...
+they are considered being scheduled, and can release changes before expected.
```
Comment (aduth) [inline _articles/appdev-deploy.md:42]: Might be clearer to elaborate on "considered being scheduled". If I'm following correctly, is it intending to communicate something along the lines of "most people will expect changes to be deployed on a schedule, so early releases can be surprising" ?

**Review by solipet — APPROVED** [review]
A couple typos, otherwise great!

**Review by aduth — APPROVED** [review]

File: _articles/appdev-deploy.md (hunk 1 above)
Comment (zachmargolis) [reply _articles/appdev-deploy.md]: "as they are" 🤦  https://github.com/18F/identity-handbook/pull/99#discussion_r592658569

File: _articles/appdev-deploy.md (hunk 3 above)
Comment (zachmargolis) [reply _articles/appdev-deploy.md]: I think I meant "as they're being planned" but your note about surprise is great, added in 791b43cda6d6087ed9fe7c549d0f44646e800e9d

**Review by stevegsa — APPROVED** [review]
EXCELLENTE!

**Comment by mitchellhenke** [comment]
Thank you!! 👏🏼 👏🏼 👏🏼

**Pull request merged**